nvflare/fuel/hci/proto.py

```diff
@@ -96,6 +96,12 @@ class InternalCommands(object):
     CHECK_SESSION = "_check_session"
     LIST_SESSIONS = "list_sessions"
 
+    commands = [PWD_LOGIN, CERT_LOGIN, LOGOUT, GET_CMD_LIST, CHECK_SESSION, LIST_SESSIONS]
```

Comment:
Is "list_sessions" internal command? is it visible to regular users?

Reply:
that's an internal command, InternalCommands.LIST_SESSIONS